app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import random
+
+from dateutil.parser import isoparse
+
+from django.contrib.gis.geos import Polygon
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.core.management.base import BaseCommand
+
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import Item
+from stac_api.validators import MEDIA_TYPES
+
+# path definition relative to the directory that contains manage.py
+logger = logging.getLogger(__name__)
+
+XMIN = 2570000
+# this would be the max extent for CH
+# XMAX = 2746000
+XMAX = 2580000
+YMIN = 1130000
+# this would be the max extent for CH
+# YMAX = 1252000
+YMAX = 1140000
+
+MIN_DATETIME = isoparse('1970-01-01T00:00:01Z')
+MAX_DATETIME = isoparse('2020-12-31T23:59:10Z')
+
+
+def random_datetime(start, end):
+    """Generate a random datetime between `start` and `end`"""
+    return start + datetime.timedelta(
+        # Get a random amount of seconds between `start` and `end`
+        seconds=random.randint(0, int((end - start).total_seconds())),
+    )
+
+
+class Command(BaseCommand):
+    help = """Populates the local test database with dummy data
```

Comment:
Maybe a more explicit help here and how to use it. For example `--clean` don't populate anything it just deletes the generated stuff. Also the command doesn't do nothing when no option is given, so maybe a check that al least 1 option is given would be good. If I understand correctly the name of the command is `perftest` however it doesn't don any performance tests, it just populate a lot of random data for performance testing, so `populate_perftestdb` would be more correct and inline with `populate_testdb`.

Reply:
well, if you wanna be strict with naming then also `populate_testdb` doesn't make sense as we don't populate the testdb but we create sample data. So more appropriate would in my opinion then be `manage.py sampledata --populate` and `manage.py dummydata --populate`.

Reply:
`app/manage.py dummydata --help` does print the help along with the arguments